scripts/gas_diff_stats.py

```diff
@@ -0,0 +1,140 @@
+"""A script to collect gas statistics and print it.
+
+Useful to summarize gas differences to semantic tests for a PR / branch.
+
+Dependencies: Parsec (https://pypi.org/project/parsec/) and Tabulate
+(https://pypi.org/project/tabulate/)
+
+  pip install parsec tabulate
+
+Run from root project dir.
+
+  python3 scripts/gas_diff_stats.py
+
+Note that the changes to semantic tests have to be committed.
+
+Assumes that there is a remote named ``origin`` pointing to the Solidity github
+repository. The changes are compared against ``origin/develop``.
```

Comment:
If anyone ever has too much time, it'd be nice to have an optional argument for this that just defaults to ``origin/develop``.